extensions/oxide-ipc/src/lib.rs

```diff
@@ -24,27 +30,19 @@ pub fn get_state_struct() -> OxideStateDto {
     }
 }
 
-/*
-pub fn wait_for_state_change() -> OxideState {
-    println!("Waiting for state change");
-    let state = async_std::task::block_on(wait_for_state_change_async()).unwrap();
-    println!("Got state change: {}", state);
-    serde_json::from_str(&state).unwrap()
-}*/
-
 pub fn state_signal_channel(sender: Arc<Mutex<Sender<OxideStateDto>>>) {
     println!("Waiting for state change");
     async_std::task::block_on(state_signal_channel_async(sender)).unwrap();
 }
 
 pub fn switch_workspace(index: usize) {
-    sent_event("gotoworkspace", Some(index.to_string()));
+    sent_event("gotoworkspace", Some(index.to_string())).ok();
 }
 
 pub fn next_workspace() {
-    sent_event("gotoworkspace", Some("next".to_string()));
+    sent_event("gotoworkspace", Some("next".to_string())).ok();
```

Comment:
I think it would be best to return results here as this makes most sense in the context of a library

Reply:
follow up ticket